tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py27, py34, py35, py36, py37, flake8, black
+toxworkdir = {env:TOX_WORK_DIR:.tox}
+
+[testenv]
+sitepackages = False
+commands = python setup.py test {posargs}
+
+[testenv:flake8]
+skip_install = True
+deps =
+    flake8
+commands =
+    flake8
+
+[testenv:black]
+skip_install = True
+deps =
+    black==18.9b0
```

Comment:
Why is this pinned to a specific black version?

Reply:
Black's formatting rules are still being iterated on between releases, so this stops an unchanged file that passed CI when it was merged failing CI later because a newer version of Black wants to reformat it (which we've experienced previously).

Reply:
Exactly, increasing this version should be part of a PR that also reformats the code to that version. This is also a good reason to tell developers to always use black through tox: you don't have to also specify the version, and you can format your code by running `tox -e black -- .`.

Reply:
> you can format your code by running `tox -e black -- .`

How convenient. I see the value in pinning, though.